projects/packages/jitm/src/class-jitm.php

```diff
@@ -137,36 +137,17 @@ public function jitm_enqueue_files() {
 		if ( $this->is_gutenberg_page() ) {
 			return;
 		}
-		$min = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
-		wp_register_style(
+
+		Assets::register_script(
 			'jetpack-jitm-css',
-			plugins_url( "css/jetpack-admin-jitm{$min}.css", __FILE__ ),
-			false,
-			self::PACKAGE_VERSION .
-			'-201243242'
-		);
-		wp_style_add_data( 'jetpack-jitm-css', 'rtl', 'replace' );
-		wp_style_add_data( 'jetpack-jitm-css', 'suffix', $min );
-		wp_enqueue_style( 'jetpack-jitm-css' );
-
-		wp_enqueue_script(
-			'jetpack-jitm-new',
-			Assets::get_file_url_for_environment( 'js/jetpack-jitm.min.js', 'js/jetpack-jitm.js', __FILE__ ),
-			array( 'jquery' ),
```

Comment:
Since our JavaScript still uses jQuery, we would have to declare it as dependency.

Reply:
I added it back in 09c1086603baeebc16fb3dbbe1b7bd5e72551e86